### DISCUSSION TITLE
Blade scene editing with Egui

### DISCUSSION THREAD

**Discussion by kvark** [emilk/egui#3530] [Show and tell]
I've been using Egui for the debug GUI as well as scene editing in [Blade](https://github.com/kvark/blade).
**Thank you for all the work on the library!**

![scene-editor](https://github.com/emilk/egui/assets/107301/7ef9587b-01ab-4c94-8556-58e6d1613339)
![scene-editor2](https://github.com/emilk/egui/assets/107301/10e1990e-8f3a-4b11-9760-17fb5e3739be)
![svgf-corset-perf](https://github.com/emilk/egui/assets/107301/db8baa96-f11d-4e83-a297-3cc3a9a7e2ce)
![local-spawn](https://github.com/emilk/egui/assets/107301/eb0e3047-d75f-4d88-87c1-ec1aa759d7cb)

Feature-wise, I'm using:
  - bunch of sliders, drag values
  - list of selectable objects
  - multiple panels
  - egui-winit for window integration
  - egui-plot for rough CPU performance tracking
  - egui-gizmo for object manipulation
  - Runs on Vulkan, Metal, and probably GLES-3

**Comment by stonerfish** [comment]
Rad.  You should put a post in the [Showcase gallery.](https://github.com/emilk/egui/issues/996)
